### PR TITLE
[Fix] Outdated Information, Grammatical Errors, Typos, Clarity & more

### PR DESCRIPTION
**Pull request by nngggaming**
This pull request includes fixes for various issues including outdated information, grammatical errors, typos, and clarity issues.

Changes:
> `README.md`
> - Fixed grammar issues
> - Fixed various typos
> - Improved clarity
> 
> `SECURITY.md`
> - Updated Supported Versions
> - Updated Discord contact information
> - Fixed grammar issues
> - Fixed tone issues
> - Improved clarity

### CODE REVIEW

**Comment by NamVr** [comment]
Sorry! I was caught up in semester exams. I will review this ASAP, I appreciate your patience :)

File: README.md [hunk 1]
```diff
@@ -108,9 +108,9 @@ To get a local/server copy up and running follow these simple example steps.
     npm run init
     ```
 
-    > You have to follow the steps in the `init` script. Put your token, bot-id, client-id, owner-id. The script will automatically create `config.json` & `database` for you. **:warning: This is one time only!**
+    > You have to follow the steps in the `init` script. Input your `token`, `client-id`, `owner-id` and `guild-id`. The script will automatically create `config.json` & `database` for you. **:warning: This is a one-time setup!**
```
Comment (NamVr) [inline README.md:111]: Could you adhere to config.json field values and change this section please?

File: SECURITY.md [hunk 2]
```diff
@@ -2,24 +2,25 @@
...
 ## Reporting a Vulnerability
 
-**Do not report any vulnerability on github**!
+**Do not report vulnerabilities on GitHub**!
 
-Please let me know immediately via emails or discord dm! <br>
-**Discord**: `allroundernaman#3915` <br>
+Please let me know immediately via email or Discord DM (Direct Message)! <br>
+**Discord**: `namanvrati` <br>
 **Email**: `info@namanvrati.cf` <br>
```
Comment (NamVr) [inline SECURITY.md:20]: If possible please change my email as well to info@namanvrati.me

**Review by NamVr — CHANGES_REQUESTED** [review]
Hi, please complete the next 2 changes to get it merged with main!

**Comment by NamVr** [comment]
PS: Sorry for late reply, got caught up in a lot of other things recently!

File: README.md (hunk 1 above)
Comment (nngggaming) [reply README.md:111]: My original PR was made to correct the discrepancies between the information provided in README.md and the config.json fields.

**Review by NamVr — DISMISSED** [review]

**Comment by NamVr** [comment]
@nngggaming I meant to say, as you can see in `config-example.json`, the field values are in `_` underscore, not `-` hyphen. If you could please correct this!

**Review by NamVr — APPROVED** [review]
LGTM

**Pull request merged**